optimum/habana/accelerate/accelerator.py

```diff
@@ -485,6 +485,12 @@ def prepare_model(self, model: torch.nn.Module, device_placement: bool = None, e
                             ),
                             auto_wrap_policy=fsdp_plugin.auto_wrap_policy,
                         )
+
+                """
+                TODO: Temporarily disable this upcast due to FSDP graph compile issue.
```

Comment:
Please add link to PR where this code was added to "accelerate" repo. Will help to understand the context when issue is debugged in future.

Reply:
Updated .